#request
bool global
bool optimized
bool graphOnly
---
#response
rtabmap/MapData data
